.github/workflows/stage-4-acceptance.yaml

```
@@ -109,16 +98,28 @@ jobs:
     name: "Integration test"
     runs-on: ubuntu-latest
     needs: environment-set-up
-    timeout-minutes: 10
     steps:
-      - name: "Checkout code"
-        uses: actions/checkout@v5
-      - name: "Run integration test"
-        run: |
-          make test-integration
-      - name: "Save result"
-        run: |
-          echo "Nothing to save"
+      - uses: actions/checkout@v5.0.0
+      # Calls out to the nhs-notify-internal repo.
+      # The nhs-notify-internal repo will run the tests
+      # setup in ./.github/actions/acceptance-tests/action.yaml
+      - name: Trigger Acceptance Tests
+        shell: bash
+        env:
+          APP_PEM_FILE: ${{ secrets.APP_PEM_FILE }}
+          APP_CLIENT_ID: ${{ secrets.APP_CLIENT_ID }}
+          TARGET_ENVIRONMENT: ${{ inputs.target_environment }}
+          TARGET_ACCOUNT_GROUP: ${{ inputs.target_account_group }}
+        run: |
+          .github/scripts/dispatch_internal_repo_workflow.sh \
+            --infraRepoName "nhs-notify-digital-letters" \
+            --overrideProjectName "nhs" \
+            --releaseVersion "${{ inputs.target_release_version || github.head_ref || github.ref_name }}" \
+            --targetWorkflow "dispatch-contextual-tests-dynamic-env.yaml" \
+            --targetEnvironment "$TARGET_ENVIRONMENT" \
+            --targetAccountGroup "$TARGET_ACCOUNT_GROUP" \
+            --targetComponent "dl" \
+            --internalRef "feature/CCM-12732_digital_letters_acceptance_tests"
```

Remove once https://github.com/NHSDigital/nhs-notify-internal/pull/462 is merged.
```suggestion
            --targetComponent "dl"
```